core/bug_api.php

```diff
@@ -544,6 +546,52 @@ function create() {
 		history_log_event_direct( $this->id, 'status', $t_original_status, $t_status );
 		history_log_event_direct( $this->id, 'handler_id', 0, $this->handler_id );
 
+		# Now that the issue is added process the @ mentions
+		$t_all_mentioned_user_ids = array();
+
+		$t_mentioned_user_ids = mention_get_users( $this->summary );
+		$t_all_mentioned_user_ids = array_merge( $t_all_mentioned_user_ids, $t_mentioned_user_ids );
+		
+		$t_mentioned_user_ids = mention_get_users( $this->description );
+		$t_all_mentioned_user_ids = array_merge( $t_all_mentioned_user_ids, $t_mentioned_user_ids );
+
+		if( !is_blank( $this->steps_to_reproduce ) ) {
+			$t_mentioned_user_ids = mention_get_users( $this->steps_to_reproduce );
+			$t_all_mentioned_user_ids = array_merge( $t_all_mentioned_user_ids, $t_mentioned_user_ids );
+		}
+
+		if( !is_blank( $this->additional_information ) ) {
+			$t_mentioned_user_ids = mention_get_users( $this->additional_information );
+			$t_all_mentioned_user_ids = array_merge( $t_all_mentioned_user_ids, $t_mentioned_user_ids );
+		}
+
+		$t_filtered_mentioned_user_ids = access_has_bug_level_filter(
+			config_get( 'view_bug_threshold' ),
+			$this->id,
+			$t_all_mentioned_user_ids );
+
+		$t_removed_mentions_user_ids = array_diff( $t_all_mentioned_user_ids, $t_filtered_mentioned_user_ids );
+
+		if( !empty( $t_all_mentioned_user_ids ) ) {
+			$t_mention_text = $this->description . "\n\n";
+
+			if( !is_blank( $this->steps_to_reproduce ) ) {
+				$t_mention_text .= lang_get( 'email_steps_to_reproduce' ) . "\n\n";
+				$t_mention_text .= $this->steps_to_reproduce . "\n\n";
+			}
+
+			if( !is_blank( $this->additional_information ) ) {
+				$t_mention_text .= lang_get( 'email_additional_information' ) . "\n\n";
+				$t_mention_text .= $this->additional_information . "\n\n";
+			}
+
+			mention_process_user_mentions(
+				$this->id,
+				$t_filtered_mentioned_user_ids,
+				$t_mention_text,
+				$t_removed_mentions_user_ids );
```

Comment:
There is no parameter for $t_removed_mentions_user_ids in function mention_process_user_mention
```php
/**
 * Process users that are mentioned on the specified bug.
 *
 * @param int $p_bug_id The bug id.
 * @param array $p_mentioned_user_ids An array of user ids
 * @param string $p_message The message containing the mentions.
 * @return void
 */
function mention_process_user_mentions( $p_bug_id, $p_mentioned_user_ids, $p_message = '' ) {
	email_user_mention( $p_bug_id, $p_mentioned_user_ids, $p_message );
}
```